server/src/site/xdoc/config-mailetcontainer.xml

```diff
@@ -27,17 +27,14 @@
 
   <section name="Mailet Container Configuration">
 
-    <p>Consult <a href="https://github.com/apache/james-project/tree/master/server/app/src/main/resources/mailetcontainer-template.xml">mailetcontainer-template.xml</a> in GIT to get some examples and hints.</p>
-    
-    <p>Changes between 3.0-M2 and 3.0-beta3:</p>
-    <ul>
-      <li>No more mailetpackage specification in mailetcontainer.xml</li>
-      <li>Exactly one standard package, hard-coded in source code</li>
-      <li>Standard package applied to mailet classnames without package prefix</li>
-      <li>All custom mailets must be specified with full package prefix</li>
-      <li>Package prefixed custom mailet names will be searched across the entire applicable class path</li>
-      <li>Composite matchers</li>
-    </ul>
+      <p>This documents explains how to configure Mail processing upon dequeue. Mails pass threw the MailetContainer. The
+          MailetContainer is composed of Mailets (execution units that perform actions based on incoming mail) and of Matchers
+          (condition for executing a mailet) arranged into processors (execution stream).</p>
```

Comment:
List of mailet/matcher pairs allowing better logical organisation of MailetContainer